less/v1/items.less

```diff
@@ -209,6 +209,88 @@
         opacity: 0.5;
         font-size: 0.75em;
       }
+      
+      // Description @Embeds
+      figure.content-embed {
+
+        margin: 0.5em 0;
+
+        // "inline" captions (non-rolltables) need some 
+        // space between the user provided
+        // caption and the caption document link border
+        strong.embed-caption {
+          margin-right: 0.3em;
+        }
+
+        .roll-table-embed {
+          // Numerical ranges in the first column of roll tables
+          // are best displayed without line breaks.
+          // Centering follows wotc styling.
+          thead tr th:first-child,
+          tbody tr td:first-child {
+            white-space:nowrap;
+            text-align: center;
+            padding-right: 2px;
+          }
+        }
+
+        // 'caption-top' is useful for embedding extended item text within
+        // another item, such as RollTables. Tweak stock formatting to look
+        // more like the surrounding parent text.
+        &.caption-top {
```

Comment:
We should probably prefer the use of the `captionPosition` option for `@Embed` but this might have to do for v11.